react/week1/todo/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './App.css';
+import ToDoList from './ToDoList';
+import ToDoItem from './ToDoItem';
+const toDoThing = [
```

Comment:
I think we could improve the naming here. 
```suggestion
const toDoList = [
```